source/data-resources.rst

```diff
@@ -46,14 +54,15 @@ These marker gene reference databases are formatted for use with QIIME 1 and QII
 Greengenes (16S rRNA)
 `````````````````````
 
-- `13_8 <ftp://greengenes.microbio.me/greengenes_release/gg_13_5/gg_13_8_otus.tar.gz>`_ (most recent)
-- `13_5 <ftp://greengenes.microbio.me/greengenes_release/gg_13_5/gg_13_5_otus.tar.gz>`_
-- `12_10 <ftp://greengenes.microbio.me/greengenes_release/gg_12_10/gg_12_10_otus.tar.gz>`_
+- `2022.10 <http://ftp.microbio.me/greengenes_release/2022.10/>`_ (most recent)
+- `13_8 <http://greengenes.microbio.me/greengenes_release/gg_13_5/gg_13_8_otus.tar.gz>`_ 
+- `13_5 <http://greengenes.microbio.me/greengenes_release/gg_13_5/gg_13_5_otus.tar.gz>`_
+- `12_10 <http://greengenes.microbio.me/greengenes_release/gg_12_10/gg_12_10_otus.tar.gz>`_
 - `February 4th, 2011 <http://greengenes.lbl.gov/Download/Sequence_Data/Fasta_data_files/Caporaso_Reference_OTUs/gg_otus_4feb2011.tgz>`_
 
-Find more information about Greengenes in the `DeSantis (2006) <http://aem.asm.org/content/72/7/5069.full>`_ and `McDonald (2012) <https://www.nature.com/articles/ismej2011139>`_ papers.
+Find more information about Greengenes in the `DeSantis (2006) <http://aem.asm.org/content/72/7/5069.full>`_, `McDonald (2012) <https://www.nature.com/articles/ismej2011139>`_, and `McDonald (2022) <https://www.biorxiv.org/content/10.1101/2022.12.19.520774v1>`_ papers.
 
-**License Information** can be found on the `Greengenes website <https://greengenes.secondgenome.com/>`_. Greengenes data are released under a `Creative Commons Attribution-ShareAlike 3.0 License <https://creativecommons.org/licenses/by-sa/3.0/deed.en_US>`_.
+**License Information** can be found on the `Greengenes website <https://greengenes.secondgenome.com/>`_. Greengenes data (prior to 2022) are released under a `Creative Commons Attribution-ShareAlike 3.0 License <https://creativecommons.org/licenses/by-sa/3.0/deed.en_US>`_. Greengenes data (2022-) are released under a `BSD-3 license <http://ftp.microbio.me/greengenes_release/2022.10/00LICENSE>`_.
```

Comment:
Should this be _Greengenes2 data (2022-) are released..._? 

Also, a more general question - do you think we should start referring to the older releases collectively as Greengenes1? That's kind of how we handle the QIIME / QIIME 1 / QIIME 2 situation now (where I typically use _QIIME_ to refer to _QIIME 1_ and _QIIME 2_).

Reply:
I don't think so. All Greengenes data (2022-) is under BSD-3. 

I'm torn on that to be honest. With Greengenes, the version numbers are pretty distinct and usually associated with the Greengenes reference. We remarked it as "Greengenes2" to reflect a major change in the database, however (to me at least) the most important thing is still the version number. 

I dunno, what do you think will be least confusing for users?

Reply:
From my perspective calling it Greengenes2 (and then having versions of that, like 2022.11) signifies fundamental changes to the product opposed to an incremental update, and given the long gap between versions I suspect calling it Greengenes2 will be less confusing (and possibly easier to sell to users and/or funders). 

Your call though - just depends on how you want to brand it. I thought it may have been a typo.

Reply:
Fair point :) I'll revise to Greengenes2 which is consistent with our other uses